Frameworks/Social.Xaml/ShareSheet.xaml

```diff
@@ -0,0 +1,20 @@
+<ContentDialog
+    x:Class="Social.Xaml.ShareSheet"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Social.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Facebook"
+    PrimaryButtonText="Post"
+    SecondaryButtonText="Cancel"
+    PrimaryButtonClick="ShareSheet_Post"
+    SecondaryButtonClick="ShareSheet_Cancel">
+
+    <StackPanel>
+        <TextBlock Text="User Name" />
+        <TextBox PlaceholderText="Say something..." x:Name="Message" />
+        <Canvas Height="100" x:Name="Preview" />
```

Comment:
We probably won't support sharing more than one link at a time, since the Facebook API doesn't provide a way to do that, at least as far as I can see. I should add checks to prevent that from happening.

Sharing multiple images at once is possible, so I should fix that. The images should appear in a horizontal row, to match the reference platform.

---
In reply to: [107997438](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107997438) [](ancestors = 107997438)